js/env.js

```diff
@@ -3,3 +3,4 @@ const nftAddress = ""; // Paste the address of the NFT collection.
 const chainId = "0x507"; // Moonbase
 // const chainId = "0x504"; // Moonbeam
 // const chainId = "0x250"; // Astar
+const useNestableNfts = true;
```

Comment:
I would use RMRK naming. Something like isRmrk

Reply:
As we decided we can only use one ABI this variable is not needed anymore and was therefore removed.